modules/disabling-insights-operator-gather.adoc

```diff
@@ -5,9 +5,13 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="disabling-insights-operator-gather_{context}"]
-= Disabling the Insights Operator gather operations
+= Disabling the Insights Operator periodic gather operations 
 
-You can disable the Insights Operator gather operations. Disabling the gather operations gives you the ability to increase privacy for your organization as Insights Operator will no longer gather and send Insights cluster reports to Red Hat. This will disable Insights analysis and recommendations for your cluster without affecting other core functions that require communication with Red Hat such as cluster transfers. You can view a list of attempted gather operations for your cluster from the `/insights-operator/gathers.json` file in your Insights Operator archive. Be aware that some gather operations only occur when certain conditions are met and might not appear in your most recent archive.
+You can optionally disable the periodic `InsightsDataGather` operations that the Insights Operator runs every 2 hours by default. Disabling the periodic data gather operations gives you the ability to increase privacy for your organization as Insights Operator will no longer gather and send Insights cluster reports to Red Hat. 
+
+Disabling gather operations will also disable Insights analysis and recommendations for your cluster without affecting other core functions that require communication with Red Hat such as cluster transfers. 
+
+You can view a list of attempted gather operations for your cluster from the `/insights-operator/gathers.json` file in your Insights Operator archive. Be aware that some gather operations only occur when certain conditions are met and might not appear in your most recent archive.
```

Comment:
Do the user need to know what certain conditions must be met for the gather operations appear in the archive?

Reply:
> Do the user need to know what certain conditions must be met for the gather operations appear in the archive?

@ragevou - Good point. This piece of information might also be better placed elsewhere too. 

@opokornyy - Could you answer @ragevou's question above? 

[If the answer increases scope, we can mark this item as a follow-up task.]

Reply:
I don't think it's necessary to add it